spec/requests/api/reports_spec.rb

```diff
@@ -85,7 +85,7 @@
 
   it "can fetch a specific result as a primary collection" do
     report = FactoryGirl.create(:miq_report_with_results)
-    result = report.miq_report_results.first
```

Comment:
Wow, this would have been confusing with the prior iteration.  :+1: 

I often use `result` as a variable(naming is hard) so using `response_hash` makes instances like this less confusing.